src/arrays.js

```diff
@@ -1,61 +1,86 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
+  const newIndex = index < array.length ? index : index - array.length;
```

Comment:
Nice work. Extra challenge - Can you think of a way you could refactor this using the `%` operator into a one liner?